Model/Smartcalcs.php

```diff
@@ -215,20 +214,12 @@ public function getTaxForOrder(
 
         if ($this->_orderChanged($order)) {
             $client = $this->clientFactory->create();
-            $client->setUri(TaxjarConfig::TAXJAR_API_URL . '/magento/taxes');
-            $client->setConfig([
-                'useragent' => $this->tjHelper->getUserAgent(),
-                'referer' => $this->tjHelper->getStoreUrl()
-            ]);
-            $client->setHeaders('Authorization', 'Bearer ' . $apiKey);
```

Comment:
@ryanreeves-taxjar We use `ZendClientFactory` directly here to pass an API token based on the store ID of the quote object:

```php
$apiKey = preg_replace('/\s+/', '', $this->scopeConfig->getValue(TaxjarConfig::TAXJAR_APIKEY,
    \Magento\Store\Model\ScopeInterface::SCOPE_STORE,
    $quote->getStoreId()
));
```

In our client class we get the API token like so:

```php
$this->apiKey = trim($this->scopeConfig->getValue(TaxjarConfig::TAXJAR_APIKEY));
```

Do you anticipate any issues here dropping this code?

Reply:
@fastdivision I ended up reverting those changes because of some additional issues I encountered during further testing.